src/ui/views/ui_app.jade

```diff
@@ -38,12 +38,13 @@ block content
         loading.removeChild(loading.lastChild);
       }
 
+      var buildNum = window.__KBN__ && window.__KBN__.buildNum;
```

Comment:
Rather than propagating the use of `window.__KBN__`, pull in the buildNum via the `chrome` module or, if that's not an option, the `metadata` module (which is essentially just an immutable wrapper around `window.__KBN__`).